src/components/tree/phyloTree/types.ts

```diff
@@ -0,0 +1,260 @@
+import { NODE_NOT_VISIBLE, NODE_VISIBLE_TO_MAP_ONLY, NODE_VISIBLE } from "../../../util/globals";
+import { PhyloTree } from "./phyloTree";
```

Comment:
yeah, this to me really argues the PhyloTree interface should be defined and exported from this file.

Reply:
Done: https://github.com/nextstrain/auspice/blob/3cf6768919650f4294a8c7a213cfa11dda5fc462/src/components/tree/phyloTree/types.ts#L211